chemicaltoolbox/chemfp/nxn_clustering.xml

```diff
@@ -50,8 +50,7 @@
             <param name="infile" ftype="fps" value="targets.fps" />
             <param name='threshold' value='0.75' />
             <param name='oformat' value='svg' />
-            <param name='output_files' value='both' />
-            <output name="image" file='NxN_Clustering_on_q.svg' ftype="svg"/>
```

Comment:
why are you removing the test for the SVG?

Reply:
Sorry, I merged before I saw this comment.

I thought it wasn't necessary to test this, but I will re-add it in a new PR.

Reply:
https://github.com/bgruening/galaxytools/pull/903